libbeat/beat/event.go

```diff
@@ -131,3 +132,28 @@ func (e *Event) SetErrorWithOption(jsonErr common.MapStr, addErrKey bool) {
 		e.Fields["error"] = jsonErr
 	}
 }
+
+// AddECSErrorMessage sets or appends to error.message, converting
+// it to an array of strings if more than one error is present.
+func (e *Event) AddECSErrorMessage(msg string) {
```

Comment:
Is @elastic/beats OK with adding this to beat.Event?

Reply:
I just realized I removed the code that needed this method from the final PR, so I'll remove it. In retrospect some kind of append(key, value) in common.MapStr looks more useful.